packages/ag-charts-enterprise/src/features/zoom/zoomScroller.ts

```diff
@@ -24,7 +24,9 @@ export class ZoomScroller {
         const sourceEvent = event.sourceEvent as WheelEvent;
 
         // Scale the zoom bounding box
-        const dir = sourceEvent.deltaY < 0 ? -1 : 1;
+        const deltaY = sourceEvent.deltaY;
+        const factor = sourceEvent.deltaMode === 0 ? 0.1 : 1;
```

Comment:
Discussed offline: Abstract this handling in the `InteractionEvent`.

Reply:
Done. This is now handled in the `InteractionManager` class